backend/common/services/cache_service.py

```
@@ -129,6 +129,69 @@ def delete(self, key: str) -> bool:
+            try:
+                value = self.redis.getdel(key)
+                if value is None:
+                    return None
+                try:
+                    return json.loads(value)
+                except (json.JSONDecodeError, TypeError):
+                    return value
+            except RedisError as retry_error:
+                logger.error(f"[REDIS] Error in getdel for {key} after reconnect: {retry_error}")
+                return None
+        except RedisError as e:
```

Ruff (0.15.7)

[warning] try-except block with duplicate exception `RedisError`

(B025)